### PR TITLE
env-specific cache-refresh

### PR DESCRIPTION
**Pull request by bhayden53**
* overrides the long refresh time on inputs and control in dev and test, primarily for developers and testers who need quick turnaround on clean-all commands in order to start data processing runs on the same datasets

### CODE REVIEW

File: lambda/refreshCacheSubmit/refresh_cache_submit.py [hunk 1]
```diff
@@ -47,3 +47,16 @@ def lambda_handler(event, context):
...
+
+
+def do_infrequent_refresh(timestamp, env):
+    # dev and test may require quick turnaround on runs and so are refreshed each time
+    override_envs = ["-dev", "-test"]
+
+    time_test = str(timestamp.minute)[0] in ("3")
+    env_test = any([s in env for s in override_envs])
+
+    if time_test or env_test:
```
Comment (jaytmiller) [inline lambda/refreshCacheSubmit/refresh_cache_submit.py:59]: Never used any(),  that's really clean!    Here you can just do "return time_test or env_test" and save a little code.

**Review by jaytmiller — APPROVED** [review]
One possible simplification but otherwise LGTM

File: lambda/refreshCacheSubmit/refresh_cache_submit.py (hunk 1 above)
Comment (bhayden53) [reply lambda/refreshCacheSubmit/refresh_cache_submit.py:59]: good call, I will make that simplification

**Comment by bhayden53** [comment]
oops, messed up the deploy script on the latest push

**Pull request merged**